### PR TITLE
Developing a Self-testing Execution Framework

### PR DESCRIPTION
**Pull request by ttnie**
Upload the developed self-test execution framework document

### CODE REVIEW

File: README_zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,438 @@
+# 开发自测试执行框架
```
Comment (tanghao-xiaomi) [inline README_zh-cn.md:1]: 入口出增加一个中英文切换按钮

Comment (ttnie) [reply README_zh-cn.md:1]: 好的，已增加

**Review by tanghao-xiaomi — APPROVED** [review]

**Review by tanghao-xiaomi — APPROVED** [review]

**Review by tanghao-xiaomi — APPROVED** [review]

**Review by GUIDINGLI — APPROVED** [review]
LGTM

**Comment by xiaoxiang781216** [comment]
@ttnie please relayout your patch to reasonable set

**Pull request closed without merging**